tests/features/search/faceting.feature

```diff
@@ -139,3 +139,17 @@ Feature: Search faceting
     And I click the `TYPE apply button`
     And I wait 2 seconds
     Then I am on page number 1
+
+  Scenario: Clear filters using using `clear all filter` button
+    When I visit the `search page`
+    And I click the `COUNTRY dropdown button`
+    And I check the "France" "COUNTRY" checkbox
+    And I click the `COUNTRY apply button`
+    And I click the `TYPE dropdown button`
+    And I check the "IMAGE" "TYPE" checkbox
+    And I click the `TYPE apply button`
+    And I wait 2 seconds
+    Then I should be on `/en/search?page=1&qf=COUNTRY%3A%22France%22&qf=TYPE%3A%22IMAGE%22&query=&view=grid`
```

Comment:
this isn't quite part of the thing we should test here. I reckon get rid of this line and move the `Then` to the last line, since that's the only expectation of this test really.